darwin/future/core/properties/get.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Optional, Union
+from uuid import UUID
+
+from pydantic import parse_obj_as
+
+from darwin.future.core.client import ClientCore
+from darwin.future.core.types.common import QueryString
+from darwin.future.data_objects.properties import FullProperty
+
+
+def get_team_properties(
+    client: ClientCore,
+    team_slug: Optional[str] = None,
+    params: Optional[QueryString] = None,
+) -> List[FullProperty]:
+    """
+    Returns a List[FullProperty] object for the specified team slug.
+
+    Parameters:
+        client (ClientCore): The client to use for the request.
+        team_slug (Optional[str]): The slug of the team to get. If not specified, the
+            default team from the client's config will be used.
+
+    Returns:
+        List[FullProperty]: List of FullProperty objects for the specified team slug.
+
+    Raises:
+        HTTPError: If the response status code is not in the 200-299 range.
+    """
+    if not team_slug:
+        team_slug = client.config.default_team
+    response = client.get(f"/v2/teams/{team_slug}/properties", query_string=params)
+    assert isinstance(response, dict)
+    return parse_obj_as(List[FullProperty], response.get("properties"))
+
+
+def get_team_full_properties(
+    client: ClientCore,
+    team_slug: Optional[str] = None,
+    params: Optional[QueryString] = None,
+) -> List[FullProperty]:
+    params = (
+        params + QueryString({"include_values": True})
+        if params and not params.get("include_values")
+        else QueryString({"include_values": True})
```

Comment:
to avoid `darwin.future.exceptions.UnprocessibleEntity: UnprocessibleEntity: <Response [422]>` error when we call:
```python
client = Client.local()
properties = get_team_full_properties(client)
```
Updating `True` -> `"true"`

```suggestion
        params + QueryString({"include_values": "true"})
        if params and not params.get("include_values")
        else QueryString({"include_values": "true"})
```

Reply:
good catch. Fixed with a change to querystring though.